test/babel/fixtures/class-properties/block-body/expected.js

```diff
@@ -0,0 +1,21 @@
+var _this = this;
+
+class Foo {
+  bar = (...params) => _this.__bar__REACT_HOT_LOADER__(...params);
```

Comment:
Just noticed this line.
Wouldn't a call to `foo.bar()` be forwarded to `window.__bar__REACT_HOT_LOADER__` instead of the class instance? Since the first line binds `_this` to the global object and not the class instance?